### ISSUE TITLE
Archive Biodiversity Projects and Networks databases

### ISSUE THREAD

**Issue by CynthiaParr-USDA** [tdwg/infrastructure#19]
Old front end tools no longer work, but Stan has gotten databases dumps.  Need to set up a Typo3 page and explain and provide links.

**Comment by CynthiaParr-USDA** [comment]
Or better yet, add the files to a new GitHub project and if someone wants to pick up on it that's great.

**Comment by peterdesmet** [comment]
I would put them as 3 files/directories in one repository on GitHub, but best to have others vote on this as well.

**Comment by mdoering** [comment]
+1 for github. The less typo3 we use the better

**Comment by pmergen** [comment]
+1 agreed. Github proved convenient to use.

**Comment by peterdesmet** [comment]
How should we (short)name that repo? In what format can the exported data be delivered (3 flat csvs)?

**Comment by CynthiaParr-USDA** [comment]
Stan, can you provide the exported data to Peter to set up the repo?
the long names are Biodiversity Information Projects of the World and Biodiversity Information Networks Database
so short name for the combined repo could be
Biodiv-projects-networks

**Comment by sblum** [comment]
Yes, I can provide those.  They were actually just tables in the MySQL DB that underpins our site, and they've been migrated over to the DB on the new site.  I think both CSV and SQL inserts should be deposited.  (I have had much less trouble getting SQL to work if values contain commas and quotes.)  Might be good to also include a separate metadata file for each table.

**Comment by peterdesmet** [comment]
@sblum, how flat are those tables? I'd prefer CSV as you can actually maintain these on GitHub, but it only makes sense if the data is rather flat. I can quite quickly just dump anything on GitHub, but it would be nice to clean it up a bit (though I'd only be able to do that early October). @mdoering care to help at that point?

**Comment by sblum** [comment]
The data underlying each "DB" is just one table.

**Comment by peterdesmet** [comment]
Ok, a CSV file would be great then (you can use a tabbed one to avoid problems with comma, but a good export program should escape with "quotes" anyway). You can email it to me.

**Comment by peterdesmet** [comment]
@sblum can you email me the CSV files?

**Comment by peterdesmet** [comment]
Got the files, planning to call the repository `biodiversity-information-initiatives`

**Comment by tucotuco** [comment]
None of the links on http://www.tdwg.org/biodiv-projects/projects-database work. Is that related?

**Comment by sblum** [comment]
John,  I'm not sure what you're referring to here.  All the individual project links I see listed at the URL you pasted DO work -- i.e., they display the record for the project.  Were you just getting broken links?

**Comment by peterdesmet** [comment]
@tucotuco: links work for me as well. All content of that DB is now archived at: https://github.com/tdwg/bii/tree/master/projects

**Comment by CynthiaParr-USDA** [comment]
Don't forget redirects.

**Comment by tucotuco** [comment]
I still the same problem, and was reporting it for others who had seen it as well. Here is a partial screen shot.
![image](https://cloud.githubusercontent.com/assets/462827/12082014/83d79c78-b269-11e5-9618-1748c54e34e2.png)

**Comment by sblum** [comment]
I can replicate the problem with Firefox, but I get the expected results
Chrome.  I'm not logged in with Chrome, so I'm puzzled.  I don't think this
is a high priority for us at the moment, but I'll try not to let it fall
off the radar entirely.

-Stan

**Comment by peterdesmet** [comment]
I cannot replicate the problem in Firefox. Since we plan to redirect anyway, we should maybe just do that? Is there a reason not to redirect for the moment?

**Comment by CynthiaParr-USDA** [comment]
I don't see any reason not to redirect. Who can set those up?
